elytron/src/test/resources/org/wildfly/extension/elytron/tls-test.xml

```diff
@@ -23,21 +23,25 @@
     </credential-stores>
     <tls>
         <key-stores>
-            <key-store name="FireflyKeystore" type="JKS">
-                <file path="firefly.keystore" relative-to="jboss.server.config.dir"/>
+            <key-store name="FireflyKeystore" >
                 <credential-reference store="test" alias="master-password-alias"/>
+                <implementation type="JKS" />
```

Comment:
No don't add any default there - I believe we have agreed to go for the JDK default if not specified.  If we set a default then it does not really make sense to split out into it's own element.